src/map/clif.c

```diff
@@ -2980,10 +2980,16 @@ static void clif_storageItems(struct map_session_data *sd, enum inventory_type t
 	int i = 0;
 	struct item_data *id;
 
+#define STORAGE_ITEMS_MAX_ITERATION 500
```

Comment:
probably better on the fly calculate max possible items per packet for current packet version?
and move this define to one of headers?

Reply:
That would be better to calculate for each packer version indeed, the reason i did not put the define in header is because i want to keep the scope of the define for this function only.

Reply:
I will rewrite the function to be more flexible.